packages/table/test/tableTests.tsx

```diff
@@ -562,13 +566,15 @@ describe("<Table>", () => {
             document.body.appendChild(containerElement);
 
             const tableComponent = ReactDOM.render(
-                <Table
-                    numRows={LARGE_NUM_ROWS}
-                    numFrozenColumns={NUM_FROZEN_COLUMNS}
-                    numFrozenRows={NUM_FROZEN_ROWS}
-                >
-                    {renderColumns({}, LARGE_NUM_COLUMNS)}
-                </Table>,
+                <div style={{ height: CONTAINER_HEIGHT, width: CONTAINER_WIDTH }}>
```

Comment:
so, what happens if users DON'T put the table in an explicitly sized div? can we never use normal flow?